stable/spinnaker/README.md

```diff
@@ -155,3 +155,11 @@ halyard:
 ```
 
 Any files added through `additionalConfigMaps` will be written to disk at `/opt/halyard/additionalConfigMaps`.
+
+### Set custom annotations for the halyard pod
+
+```yaml
+halyard:
+  additionalAnnotations:
```

Comment:
What about just calling it `halyard.annotations` instead of `halyard.additionalAnnotations` since its the only ones that get added?

Reply:
Sounds fair. I made the change.